src/MinedMap.cpp

```diff
@@ -195,7 +195,12 @@ static int64_t getModTime(const std::string &file) {
 #ifdef _WIN32
 	return (int64_t)s.st_mtime * 1000000;
 #else
-	return (int64_t)s.st_mtim.tv_sec * 1000000 + s.st_mtim.tv_nsec / 1000;
+    #ifdef __APPLE__
```

Comment:
I'd prefer to use `#elif defined(...)` like it was done in #19 to avoid increasing the nesting depth.

Reply:
That is cleaner. Made the change.